### ISSUE TITLE
Dolby Vision Profile 7 fallback

### ISSUE THREAD

**Issue by realFPS** [jarnedemeulemeester/findroid#868]
**Is your feature request related to a problem? Please describe.**
Remux with Dolby Vision profile 7 cannot be played as HDR with Exoplayer.

**Describe the solution you'd like**
Allow playing DV profile 7 files as HDR on compatible devices. An open source video player called Just Player also uses Exoplayer but it can do this, so why not Findroid.
![SmartSelect_20241026_000642_Just Player.png](https://github.com/user-attachments/assets/60ac719b-ed4d-4751-82d4-27ead013ccd2)

**Comment by MrPeteypoo** [comment]
I noticed this too, using MPV as a player gets around this problem.

**Comment by realFPS** [comment]
That is not a solution because android mpv does not support HDR and is tone mapping and displaying it as SDR.

**Comment by MrPeteypoo** [comment]
Yes that's true, it allows you to watch the video but you only in SDR.

**Comment by foux** [comment]
I agree, the fallback should be set, there's no reasons not to

**Comment by AndyPro720** [comment]
I agree, what could be a workaround for now though? I can't watch DV without mkv and HDR with.

**Comment by ok626** [comment]
Moonfin has this now. Also Nuvio

**Comment by ok626** [comment]
Well those 2 actually step it up and convert to profile 8 DV

**Comment by foux** [comment]
Arnage too
